### PR TITLE
trivial build fix

### PR DESCRIPTION
**Pull request by amosbird**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Non-significant (changelog entry is not needed)

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):

...

Detailed description (optional):

...

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]
Ok. But we don't see this in our CI. What version of gcc do you use?
What variable is "maybe uninitialized"?

File: dbms/src/Functions/GeoUtils.h [hunk 1]
```diff
@@ -261,13 +261,18 @@ void PointInPolygonWithGrid<CoordinateType>::buildGrid()
 
     for (size_t row = 0; row < grid_size; ++row)
     {
+#pragma GCC diagnostic push
```
Comment (alexey-milovidov) [inline dbms/src/Functions/GeoUtils.h]: But this is cold code. You can initialize what you want event if it's unneeded.

Comment (alexey-milovidov) [reply dbms/src/Functions/GeoUtils.h]: Just write `Box box{}`?

**Comment by amosbird** [comment]
it's gcc (Gentoo 9.2.0-r2 p3) 9.2.0 and it happens after I rebased to latest master. The fix is similar to what we did in https://github.com/yandex/ClickHouse/blob/master/dbms/src/Functions/GeoUtils.h#L227

**Pull request merged**